lib/cc/formatters/snapshot_formatter.rb

```diff
@@ -59,6 +59,10 @@ def initialize(payload)
         @improved_constants_payload = data.merge("constants" => improved_constants) if improved_constants.any?
       end
 
+      def changed?
+        alert_constants_payload || improved_constants_payload
```

Comment:
What do you think of wrapping this line in `!!()` so that the method actually returns a boolean value (as suggested by its signature) rather than either payload?